{"uuid": "6a268a66-c86f-4eb3-aa07-2ed32ca18f2b", "children": ["1a3f912c-6a0f-4081-b463-e54a6f60813b"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1715587628623, "stop": 1715587628623}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep'\n", "trace": "  File \"C:\\Users\\sharath.ls\\OneDrive - Feuji Software Solutions Pvt Ltd\\Documents\\Python\\PytestSelenium\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\sharath.ls\\OneDrive - Feuji Software Solutions Pvt Ltd\\Documents\\Python\\PytestSelenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\sharath.ls\\OneDrive - Feuji Software Solutions Pvt Ltd\\Documents\\Python\\PytestSelenium\\Tests\\configtest.py\", line 366, in log_on_failure\n    if item.rep.call.failed:\n       ^^^^^^^^\n"}, "start": 1715587628627, "stop": 1715587628631}], "start": 1715587628623, "stop": 1715587628631}